istioctl/pkg/install/k8sversion/version_test.go

```diff
@@ -152,3 +157,51 @@ func TestExtractKubernetesVersion(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
I would add a comment in here (and maybe in the version.go) that IsK8VersionSupported only returns an error if it can't parse the version and only outputs a log if the version isn't supported (not sure if there is a way to verify if the text was output, but would be a good check). My first thought looking at this test is that the 1.19 checks should be different than the 1.17 checks (1.19 being good and 1.17 being bad), but the case data looks the same.